### DISCUSSION TITLE
Matteo Pachera Paper Problem

### DISCUSSION THREAD

**Discussion by appheadspace** [firemodels/fds#13679] [Q&A] [answered]
Hi, I'm trying to solve 2D natural convection in FDS but temp near hot and cold walls are oscillating near ambient temp only. I have also added fine mesh near walls. Kindly help me. 

&HEAD CHID='chid' TITLE='title' /

&MESH ID='Full Mesh', 		IJK=25,1,25,  XB=0,0.750,	  0,0.0001,	0,0.750 /
&MESH ID='Left Fine Mesh', 	IJK=15,1,375, XB=0,0.015,	  0,0.0001,	0,0.750 /
&MESH ID='Right Fine Mesh', IJK=15,1,375, XB=0.735,0.75,  0,0.0001,	0,0.750 /
&MESH ID='Bottom Fine Mesh',IJK=375,1,15, XB=0,0.75,	  0,0.0001,	0,0.015 /
&MESH ID='Top Fine Mesh',	IJK=375,1,15, XB=0,0.75,	  0,0.0001,	0.735,0.750 /

&TIME T_END=100./

&DUMP DT_DEVC=0.05, DT_RESTART = 100.	/

&MISC STRATIFICATION=F, SIMULATION_MODE='LES', TMPA = 30./

&RADI RADIATION=.TRUE./

&SPEC ID='LJ AIR', BACKGROUND=.TRUE./

&OBST XB=0,0,0,0.0001,0,0.750 , 		SURF_ID='hot_wall'  / left
&OBST XB=0.750,0.750,0,0.0001,0,0.750, 	SURF_ID='cold_wall' / right
&OBST XB=0,0.750,0,0.0001,0,0 , 		SURF_ID='insulated' / bottom
&OBST XB=0,0.750,0,0.0001,0.750,0.750, 	SURF_ID='insulated' / top

&SURF ID = 'hot_wall', 		COLOR = 'RED',  	TMP_FRONT = 50, 	TAU_T=0., EMISSIVITY=0.9  /
&SURF ID = 'cold_wall', 	COLOR = 'BLUE',  	TMP_FRONT = 10, 	TAU_T=0., EMISSIVITY=0.9 /
&SURF ID = 'insulated', 	COLOR = 'GRAY', 	HEAT_TRANSFER_COEFFICIENT=0./

&INIT XB=0,0.75,-0.375,0.375,0,0.75, 	TEMPERATURE=30./

&DEVC XYZ=0,     0,0.375, 			IOR=1, 	QUANTITY='WALL TEMPERATURE', ID='HotWall_Temp'/
&DEVC XYZ=0.005,0,0.375, 			IOR=1, 	QUANTITY='TEMPERATURE', 	 ID='HotWall_Temp1'/
&DEVC XYZ=0.015,0,0.375, 			IOR=1, 	QUANTITY='TEMPERATURE', 	 ID='HotWall_Temp2'/
&DEVC XYZ=0.025,0,0.375, 			IOR=1, 	QUANTITY='TEMPERATURE', 	 ID='HotWall_Temp3'/

&DEVC XYZ=0.75, 0,0.375, 			IOR=-1, 	QUANTITY='WALL TEMPERATURE', ID='ColdWall_Temp'/
&DEVC XYZ=0.745,0,0.375, 			IOR=-1, 	QUANTITY='TEMPERATURE', 	 ID='ColdWall_Temp1'/
&DEVC XYZ=0.735,0,0.375, 			IOR=-1, 	QUANTITY='TEMPERATURE', 	 ID='ColdWall_Temp2'/
&DEVC XYZ=0.725,0,0.375, 			IOR=-1, 	QUANTITY='TEMPERATURE', 	 ID='ColdWall_Temp3'/

&SLCF QUANTITY='TEMPERATURE', ID='SliceY', XB=0.000,0.75,0,0,0.000,0.75/
&TAIL /

**Comment by drjfloyd** [comment]
6.3.4 Mesh Alignment

In general, there is little advantage to overlapping meshes because information is only exchanged at
exterior boundaries. This means that a mesh that is completely embedded within another receives information
at its exterior boundary, but the larger mesh receives no information from the mesh embedded
within. Essentially, the larger, usually coarser, mesh is doing its own simulation of the scenario and
is not affected by the smaller, usually finer, mesh embedded within it. Details within the fine mesh,
especially related to fire growth and spread, may not be picked up by the coarse mesh. In such cases,
it is preferable to isolate the detailed fire behavior within one mesh, and position coarser meshes at the
exterior boundary of the fine mesh. Then the fine and coarse meshes mutually exchange information.

Have you searched the extensive literature on natural convection in cavities? What does the literature say you should expect for the boundary layer thickness for your cavity and wall temperature difference?  i.e., is it reasonable to expect that 5 mm away from the wall at the midheight lies within the boundary layer?

**Reply by appheadspace** [reply]
<img width="317" alt="image" src="https://github.com/user-attachments/assets/9b12e601-cab3-4739-8e47-3962e8a1109a">

Thanks for ur reply! This is the actual experiment temp profile. X = horizontal = length = 0.75m, Y= vertical = height = 0.75m. I'm expecting similar profile. I'm new to FDS. Kindly help if I'm missing anything.

**Reply by drjfloyd** [reply, marked as answer]
Look carefully at the data you are referencing.  Matteo's paper references https://doi.org/10.1016/S0017-9310(03)00147-9 as the source of experimental data.

Your first measurement point is x=0.005 m.  With a 0.75 m wide domain this is a normalized x value of 0.005/0.75 = 0.0067.   In Table 2 of this paper, the normalized average temperature at a normalized location of (0.0067,0.5) is 0.6793 or 10 C + 0.6793*(50 C-10 C) = 37 C.  At 0.015 m or normalized x of 0.02 the normalized average temperature is 0.5270 or 31 C.

**Comment by appheadspace** [comment]
I got it now. Thank u so much @drjfloyd  !!!
